src/main/frontend/handler.cljs

```diff
@@ -128,10 +128,13 @@
                                (fn []
                                  (js/console.error "Failed to request GitHub app tokens."))))
 
-                            (watch-for-date!)
+                            (when (state/enable-journals? (state/get-current-repo))
```

Comment:
Will watch enabled when switching from a non-journal graph to a journal-enabled graph

Reply:
Yes on my test.